wpcli-vulnerability-scanner.php

```diff
@@ -12,27 +12,23 @@
  * License:           MIT
  * License URI:       https://spdx.org/licenses/MIT.html
  * Update URI:        https://github.com/10up/wpcli-vulnerability-scanner
- *
  */
+
 if ( ! defined( 'WP_CLI' ) ) {
 	return;
 }
 
-// API URL constant defined so we can override it if needed.
-if ( ! defined( 'VULN_API_URL' ) ) {
-	define( 'VULN_API_URL', 'https://wpvulndb.com/api/v3/' );
-}
+// Include Vulnerability Scanner API service classes.
+require_once __DIR__ . '/includes/class-vuln-service.php';
+require_once __DIR__ . '/includes/class-vuln-wpscan-service.php';
+require_once __DIR__ . '/includes/class-vuln-patchstack-service.php';
 
+if ( ! class_exists( 'Vulnerability_CLI' ) ) :
```

Comment:
I don't necessarily love wrapping an entire file in a conditional. Could we instead return early if the class does exist?

Reply:
@dkotter mind applying updates for this so I can merge in / release by end of week?

Reply:
Done